src/impy/common.py

```diff
@@ -368,67 +434,6 @@ def __getnewargs__(self):
         return (EventData,)
 
 
-# =========================================================================
```

Comment:
I also would like a way to change the behavior of a model, e.g. to run Pythia8 to make top events instead of min-bias events. However, we can achieve this either by adding methods to the Model (if Model specific) or on the base class MCRun (if generic, like set_stable), or if the setup has to happen before model initialization, we can pass those as options to the `__init__` of the model. Either way, I think we do not need a Settings class.

Reply:
Agree